message MSG_CLIENT_DM_GROUP_DESCRIPTION_GET
{
    optional string token = 1;
}

message MSG_DM_CLIENT_GROUP_DESCRIPTION_GET_ACK
{
    optional int32 statusCode = 1;
    message GROUP_INFO
    {
        optional string name = 1;
        optional string description = 2;
    }
    repeated GROUP_INFO groupInfo = 2;
}